WorkManagerSample/app/src/main/res/layout/activity_filter.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  ~ Copyright 2018 The Android Open Source Project
```

Comment:
2021 instead of 2018 😬

Reply:
It's been there since 2018. I'm okay with keeping it that way.